bot/cogs/metrics.py

```diff
@@ -31,8 +31,8 @@ def __init__(self, bot: Bot) -> None:
         )
         self.command_completions = Counter(
             name=f'{self.PREFIX}_command_completions',
-            documentation="Completed commands by command by guild.",
-            labelnames=('guild_id', 'command')
+            documentation="Completed commands by command by user by guild.",
```

Comment:
this string is difficult to understand. Can we try to reduce the number of consecutive `by`'s a little? Surely there must be a better way to phrase this.

Reply:
How about 99088f1?

Reply:
yeah that looks good :+1: